packs/bitbucket/rules/post_receive_webhook.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: "post_receive_webhook"
+pack: "bitbucket"
+description: "Trigger ST2 pack deploy for commit on Git branch"
+enabled: true
```

Comment:
I've disabled the rule as suggested. 

I think putting it in the packs pack does not make great sense to me as it would then be part of every install of StackStorm when it's only needed by those using BitBucket.

May be we need to pack-requirements.txt to document cross pack requirements?